packages/frontend/src/pages/Signup.tsx

```diff
@@ -178,6 +179,9 @@ const Signup: FC = () => {
 
     return (
         <Page isFullHeight>
+            <Helmet>
+                <title>Sign up - Lightdash</title>
```

Comment:
This is for registering with an invite code, I think we can call this `Register` as well 
![image](https://user-images.githubusercontent.com/1983672/206494940-ac68dfb8-3d68-4a25-9597-fbf0a10a1cc0.png)